src/main/resources/static/templates/CSRFVulnerability/LEVEL_1/CSRFVulnerability.js

```
@@ -0,0 +1,38 @@
+function addingEventListenerToSendButton() {
+  document.getElementById("sendBtn").addEventListener("click", function () {
+    let url = getUrlForVulnerabilityLevel();
+    doGetAjaxCall(
+      CallbackResponse,
+      url +
+        "?id=" +
+        document.getElementById("id").value +
+        "&pass=" +
+        document.getElementById("pass").value +
+        "&newpass=",
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)